### ISSUE TITLE
DEFAULT.QC_Accessibility_Eservices - Defect # 509 - screen reader help text

### ISSUE THREAD

**Issue by marklunn** [epimorphics/ukhpi#81]
Description: 
While the style sheets are still disabled, 'select' Structure then fieldsets/labels. Visually check any date input fields contain the screen reader help text for the date format input.

Expected: 
Date format input is part of the label of the date field on the page

Actual: 
none of the date fields contain screen reader help text for the date

**Comment by marklunn** [comment]
![bug_509_clb_picture_142](https://cloud.githubusercontent.com/assets/13219344/17770415/5b8418ee-6535-11e6-8175-519c14c4c2c6.jpeg)

**Comment by ijdickinson** [comment]
@marklunn Can you please provide an example of what the expected `screen reader help text` should be? The report says what's missing, not what you're expecting.

**Comment by marklunn** [comment]
E.g. where there is just an asterisk it will also add the word 'Required' before the input field.  It is usually on input fields and gives an extra bit of assistance to the user.  Essentially 'help text' to ensure the user knows what to do.

**Comment by MalcolmShazell** [comment]
closed - re-design has overtaken this
